backend/auth.py

```diff
@@ -14,35 +14,44 @@
 
 
 def get_token_auth_header(header):
-    """Obtains the Access Token from the Authorization Header
+    """Obtains the token from a given header
     """
-    auth = request.headers.get(header, None)
-    if not auth:
-        return Response(json.dumps({'error': 'Authorization_header_missing'}), 401)
 
-    parts = auth.split()
+    headerString = ""
```

Comment:
It's not nessecary to define the variable before, since it raises an exeption anyways